src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisShardSyncer.java

```diff
@@ -602,6 +602,7 @@ private void cleanupGarbageLeases(List<Shard> shards, List<KinesisClientLease> t
         if (!garbageLeases.isEmpty()) {
             LOG.info("Found " + garbageLeases.size() + " candidate leases for cleanup. Refreshing list of"
                     + " Kinesis shards to pick up recent/latest shards");
+            // TODO: Update to ListShardsWithFilter once lease GC is finalized
```

Comment:
Don't need this TODO, lease cleanup will be added to shutdown instead of shardsync

Reply:
Will remove